text/0118-search-api.md

```diff
@@ -108,156 +111,273 @@ It returns the number of documents matching the current search query for each gi
+Defines document attributes to be cropped. Cropped attributes have their values shortened around query terms.
+
+The number of words contained in the cropped value is defined by the `cropLength` parameter. See [3.1.1.12. `cropLength`](#31112-croplength) section.
+
+The value of `cropLength` can be customized per attribute. See [3.1.11.1. Custom `cropLength` Defined Per Cropped Attribute](#31111-custom-croplength-defined-per-attribute) section.
 
-When this parameter is set, returned documents include a `_formatted` object containing the cropped terms.
+The engine adds a marker by default in front of and/or behind the part selected by the cropper. This marker is customizable. See [3.1.1.13. `cropMarker`](#31113-cropmarker) section.
 
-Optionally, indicating a custom crop length for any of the listed attributes is possible: `attributesToCrop=["attributeNameA:25", "attributeNameB:150"]`. The custom crop length set in this way has priority over the `cropLength` parameter.
+Search results include a `_formatted` object containing the cropped attributes representation when this parameter is defined. See [3.2.1.1.2. `_formatted`](#32112-formatted) section.
 
-Instead of supplying individual attributes, it is possible to provide `["*"]` as a value: `attributesToCrop=["*"]`. This will crop the values of all attributes present in `attributesToRetrieve`.
+If `"*"` is provided as a value: `attributesToCrop=["*"]` all the attributes present in `attributesToRetrieve` will be automatically assigned to `attributesToCrop`.
 
-- 🔴 Sending a value with a different type than `Array[String]`(POST), `String`(GET) or `null` for `attributesToCrop` will return a [bad_request](0061-error-format-and-definitions.md#bad_request) error.
+- 🔴 Sending a value with a different type than `Array[String]`(POST), `String`(GET) or `null` for `attributesToCrop` returns a [bad_request](0061-error-format-and-definitions.md#bad_request) error.
 
-> See [_Formatted Field Behavior](0039-_formatted-field-behavior_.md)
+##### 3.1.11.1. Custom `cropLength` Defined Per Attribute.
 
-##### 1.2.1.10 `cropLength`
+Optionally, indicating a custom crop length for any of the listed attributes is possible:
+
+`"attributesToCrop":["attributeNameA:15", "attributeNameB:30"]`
+
+A custom crop length set in this way has priority over the `cropLength` parameter.
+
+##### 3.1.11.2. searchableAttributes
+
+As for `attributesToHighlight`, attributes not defined in the `searchableAttributes` index setting are also cropped if assigned to `attributesToCrop`.
+
+#### 3.1.1.12. `cropLength`
 
 - Type: Integer
 - Required: False
-- Default: `200`
+- Default: `10`
 
-Configures the number of characters to keep on each side of the matching query term when using the `attributesToCrop` parameter.
+Sets the total number of **words** to keep around the matched part of an attribute specified in the `attributesToCrop` parameter.
```

Comment:
I have clarified the sentence (so that it is clearer and does not leave room for doubts) and you can find examples further in the specification. See 3.1.12.1.1. Extending around section.